manifests/oci-cloud-controller-manager-rbac.yaml

```diff
@@ -50,12 +50,26 @@ rules:
   - ""
   resources:
   - endpoints
+  resourceNames:
+  - "cloud-controller-manager"
   verbs:
   - create
   - get
   - list
   - watch
   - update
+
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  resourceNames:
+  - "cloud-controller-manager"
```

Comment:
You can't limit the name of a resource during create. It's not available to RBAC since it's a part of the body not the path. 

You'll need to separate things here. One for creating configmaps and the other for get/update for the named configmap.

Reply:
Same goes for endpoints.

Reply:
Ah; that makes perfect sense. Thanks!